Missionframework/scripts/server/init_server.sqf

```diff
@@ -126,26 +126,3 @@ execVM "scripts\server\offloading\group_diag.sqf";
 if (KP_liberation_restart > 0) then {
     execVM "scripts\server\game\server_restart.sqf";
 };
-
-if (KP_liberation_limited_zeus) then {
-    zm1 setVariable ["Addons", 0, true];
-    removeAllCuratorAddons zm1;
-
-    zm1 setCuratorCoef ["edit", -1e8];
-    zm1 setCuratorCoef ["place", -1e8];
-    zm1 setCuratorCoef ["synchronize", 0];
-    zm1 setCuratorCoef ["delete", 0];
-    zm1 setCuratorCoef ["destroy", -1e8];
-} else {
-    zm1 setVariable ["Addons", 3, true];
-    removeAllCuratorAddons zm1;
-
-    private _allAddons = ("true" configClasses (configFile >> "CfgPatches")) apply {configName _x};
-    zm1 addCuratorAddons _allAddons;
```

Comment:
This isn't needed anymore in `KPLIB_fnc_initCuratorHandlers`?

Reply:
Should not be.

`_zeus setVariable ["Addons", 0, true];` or `_zeus setVariable ["Addons", 0, true];`